.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+**Which issue is resolved by this Pull Request:**
+Resolves #
+
+**Description of your changes:**
+
+
+**Testing:**
```

Comment:
We should add a section for details on manual testing completed. Also a section for if tests were added for the new feature.